src/components/Answer.jsx

```diff
@@ -0,0 +1,36 @@
+export function Ans({ correct, text, click, picked, check }) {
```

Comment:
`Answer` would be great name for this component.

Reply:
That was the old version of the component. I forgot to remove it; now it is removed.